nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/ListHDFS.java

```diff
@@ -74,7 +74,9 @@
         + "cases where files with the same timestamp are written immediately before and after a single execution of the processor. For each file that is "
         + "listed in HDFS, this processor creates a FlowFile that represents the HDFS file to be fetched in conjunction with FetchHDFS. This Processor is "
         +  "designed to run on Primary Node only in a cluster. If the primary node changes, the new Primary Node will pick up where the previous node left "
-        +  "off without duplicating all of the data. Unlike GetHDFS, this Processor does not delete any data from HDFS.")
+        +  "off without duplicating all of the data. Unlike GetHDFS, this Processor does not delete any data from HDFS."
+        +  "If the execution is triggered by an input flowfile, it's attribute will be transferred to the generated flowfiles, and it will be rooted to "
```

Comment:
```suggestion
        +  "If the execution is triggered by an input FlowFile, its attributes will be transferred to the generated flowfiles, and it will be routed to "
```